apis/r/NAMESPACE

```diff
@@ -36,6 +37,9 @@ export(ScalarMap)
 export(TileDBArray)
 export(TileDBGroup)
 export(TileDBObject)
+export(VirtualCollection)
```

Comment:
tiledbsoma-py and tiledbsoma-r aren't identical (different languages after all) but i don't see any value-add in calling these ephemeral in python and virtual in r ...